src/web/arr/trove/chart.arr

```diff
@@ -172,14 +193,14 @@ fun prep-axis(values :: P.LoN) -> {Number; Number}:
   {max-positive-height; max-negative-height}
 end
 
-fun multi-prep-axis(is-stacked :: String, value-lists :: P.LoLoN) 
+fun multi-prep-axis(is-stacked :: StackType, value-lists :: P.LoLoN) 
```

Comment:
`is-stacked` sounds like a boolean flag.  This parameter should be named `stack-type` or something.

Reply:
Yeah sorry about that, the parameter for google charts was is-stacked so I just copied that over. Changed it to stack-type!